grc/core/params/param.py

```diff
@@ -423,11 +424,12 @@ def collision_detection(row, col, row_span, col_span):
 
             if self.parent_flowgraph.get_option('output_language') == 'python':
                 widget_str = textwrap.dedent("""
-                    self.{layout}.addWidget({widget}, {row}, {col}, {row_span}, {col_span})
-                    for r in range({row}, {row_end}):
-                        self.{layout}.setRowStretch(r, 1)
-                    for c in range({col}, {col_end}):
-                        self.{layout}.setColumnStretch(c, 1)
+                    if not os.path.exists(file_path):
```

Comment:
It's more modern to use f-strings instead of `.format()` (I know much of the rest of the codebase uses `.format()`, but I think we should work to use modern Python techniques where possible!)

Reply:
seconding that; a lot of the Python you find in-tree is from a Python 2.7-heritage (as that was all we had in GNU Radio 3.7), and we instantly bumped the requirement to Python 3.6 when we released GNU Radio 3.8, exactly because we wanted f-strings!

Reply:
Yeah I've been trying to work through linter errors/warnings from rough and have found a lot of 2.7-migrated-to-3.1+ style code :P